packages/date-picker/test/overlay.test.js

```diff
@@ -17,6 +17,23 @@ function waitUntilScrolledTo(overlay, date, callback) {
   }
 }
 
+async function customizeFixture({ initialPosition, monthScrollerItems, monthScrollerOffset }) {
+  const overlay = fixtureSync(`
+        <vaadin-date-picker-overlay-content></vaadin-date-picker-overlay-content>
+      `);
+  const monthScroller = overlay.$.monthScroller;
+  monthScroller.style.setProperty('--vaadin-infinite-scroller-buffer-offset', monthScrollerOffset);
+  monthScroller.updateStyles({ '--vaadin-infinite-scroller-buffer-offset': monthScrollerOffset });
```

Comment:
This isn't needed in V24 because `updateStyles()` is only for polyfilled browsers like IE11.

Reply:
Done